definitions/private-law/json/State.json

```diff
@@ -6,7 +6,7 @@
     "Name": "Draft",
     "Description": "Draft",
     "DisplayOrder": 1,
-    "TitleDisplay": "# #${[CASE_REFERENCE]}\n ## Case Name: ${applicantCaseName}\n ## Family Man ID: ${familymanCaseNumber}"
+    "TitleDisplay": "## ${applicantCaseName}\n ## Familyman ID: ${familymanCaseNumber}\n ## Casenumber: ${[CASE_REFERENCE]}"
```

Comment:
Will this change make case number to be displayed twice in the page?

Reply:
no it has not